src/test/config/server.config.ts

```diff
@@ -0,0 +1,26 @@
+import dotenv from "dotenv";
+import chaiHttp from "chai-http";
+import chai, { expect } from "chai";
+import { StatusCodes } from "http-status-codes";
+import mongoose from "mongoose";
+import faker from "faker";
+import { User } from "../../models/user.model";
+import { Recipe, RecipeDocument } from "../../models/recipe.model";
+dotenv.config();
+process.env.MONGODB_CONNECTION_STRING = "mongodb://localhost:27017/recipe-test";
+
+import app from "../../app";
```

Comment:
Is it important that app import is after dotenv config? Just asking, it may be important